pkg/controllers/management/auth/globalroles/enqueue_test.go

```diff
@@ -9,13 +9,27 @@ import (
 	"github.com/rancher/wrangler/pkg/generic/fake"
 	"github.com/rancher/wrangler/pkg/relatedresource"
 	"github.com/stretchr/testify/require"
+	v1 "k8s.io/api/rbac/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
+func Test_grbGrIndexer(t *testing.T) {
```

Comment:
I added this test to get 100% code coverage of `enqueue.go`. I didn't make or modify this function